wiki/History_of_osu!/April_fools/en.md

```diff
@@ -0,0 +1,324 @@
+---
```

Comment:
Folder name should be `April_Fools`, as it is a proper noun.

Note: there is one img already in `April_Fools`.

Reply:
Hmm, that's a little weird, because folder is already capitalised. Not sure why its displaying as lowercase here

Reply:
Are you talking about your local repo? Try doing `git config core.ignorecase false` in your local repo dir.

Reply:
seemed to work, thank you!